Wrath/APLs/PaladinRetribution.simc

```diff
@@ -10,7 +10,7 @@ actions.precombat+=/blessing_of_might,if=!up&settings.maintain_blessing&assigned
 actions.precombat+=/blessing_of_sanctuary,if=!up&settings.maintain_blessing&assigned_blessing.blessing_of_sanctuary
 actions.precombat+=/blessing_of_wisdom,if=!up&settings.maintain_blessing&assigned_blessing.blessing_of_wisdom
 
-actions+=/hand_of_reckoning,use_off_gcd=1,if=glyph.reckoning.enabled&(target.is_boss|debuff.training_dummy.up)
+actions+=/hand_of_reckoning,use_off_gcd=1,if=!settings.hor_macros & glyph.reckoning.enabled & (debuff.training_dummy.up  |!aggro )
```

Comment:
- Remove settings.hor_macros in favor of the player using the out of the box Ability toggle

Reply:
Spaces are bad in .simc files because my importer is crude.  This one might not be busted, but best practice is to keep APLs condensed here.